### ISSUE TITLE
Add logs for the database restoration process

### ISSUE THREAD

**Issue by jstebenne** [hargata/lubelog#404]
I am migrating my Lubelog instance from UnRaid to my Kubernetes cluster and I had an issue when importing into the new instance. I found that my file limit was way too low in my own configuration (in the Nginx ingress controller), but I had no indication of it in the container's logs.

It would be nice to have more information available by default in the container logs in general, but I think it should go as far as "forced" debug logs when importing a database, so that an administrator has all the required information when doing that kind of activity.

**Comment by hargata** [comment]
Is this when you're restoring the backup?

**Comment by jstebenne** [comment]
Yes

**Comment by hargata** [comment]
@jstebenne we can't really log that since the request is blocked by NGINX before it even makes it to the backend. NGINX is the one that forces a 413 request entity too large error. The backend which handles logging won't even see the request comes in, this is why the front end will show a toast saying "check file size" when the request fails.

**Comment by jstebenne** [comment]
Oh yeah, I get that, but I would have expected some logging that could indicate that some error occurred, maybe something like "Backup restore process started" and then "Backup restore process failed". It could probably even be in the browser's console since that's the only place with reliable information.

The thing I forgot to mention, is that on my first try, I got stuck on the loading overlay, with no error displayed anywhere. My first thought was that the button was just a placeholder

**Comment by hargata** [comment]
hm, that is weird, I just tried it on the demo site and I was able to see the toast:

![image](https://github.com/hargata/lubelog/assets/155338622/0fb44197-fca9-49c3-ad89-ed80fa72e76c)

Regardless, I can just add a few lines of logging for the db restore method in the browser's log. It'll be released tomorrow in 1.2.7

**Comment by jstebenne** [comment]
Awesome! Thanks! I'll make sure to test it on my side too

**Comment by hargata** [comment]
@jstebenne that is now released in 1.2.7, the browser logs will show as below:

![image](https://github.com/hargata/lubelog/assets/155338622/d0b6f8e4-fefe-4365-bdc4-3762d72318c8)
